src/test/java/bio/terra/workspace/db/DataReferenceDaoTest.java

```diff
@@ -177,7 +230,7 @@ public void enumerateWorkspaceReferences() throws Exception {
     dataReferenceDao.createDataReference(
         secondReferenceId,
         workspaceId,
-        name,
+        name + "2",
```

Comment:
Why the "2"?

Reply:
Since uniqueness between `(workspaceId, referenceType, name)` is now enforced, this one clashed with the reference created above it